redbot/cogs/trivia/trivia.py

```diff
@@ -387,6 +389,68 @@ async def trivia_list(self, ctx: commands.Context):
             else:
                 await ctx.send(msg)
 
+    @trivia.command(name="info")
+    async def trivia_info(self, ctx: commands.Context, category: str.lower):
+        """Get information about a trivia category."""
+        try:
+            data = self.get_trivia_list(category)
+        except FileNotFoundError:
+            return await ctx.send(
+                _(
+                    "Category {name} does not exist."
+                    " See {command} for the list of available trivia categories."
+                ).format(name=inline(category), command=inline(f"{ctx.clean_prefix}trivia list"))
+            )
+        except InvalidListError:
+            return await ctx.send(
+                _(
+                    "There was an error parsing the trivia list for the {name} category."
+                    " It may be formatted incorrectly."
+                ).format(name=inline(category))
+            )
+
+        config_overrides = data.pop("CONFIG", None)
+
+        embed = discord.Embed(
+            title=_('"{category}" Category Details').format(category=category),
+            color=await ctx.embed_colour(),
+        )
+        embed.add_field(
+            name=_("Authors"), value=data.pop("AUTHOR", "").strip() or italics(_("Not provided."))
+        )
+        embed.add_field(name=_("Question count"), value=len(data))
+        embed.add_field(
+            name=_("Custom"),
+            value=category in [p.resolve().stem for p in cog_data_path(self).glob("*.yaml")],
+        )
+        embed.add_field(
+            name=_("Description"),
+            value=(
+                data.pop("DESCRIPTION", "").strip()
+                or italics(_("No description provided for this category."))
+            ),
+            inline=False,
+        )
+
+        if config_overrides:
+            msg = _(
+                "Bot gains points: {bot_plays}\n"
+                "Answer time limit: {delay} seconds\n"
+                "Lack of response timeout: {timeout} seconds\n"
+                "Points to win: {max_score}\n"
+                "Reveal answer on timeout: {reveal_answer}\n"
+                "Payout multiplier: {payout_multiplier}\n"
+                "Allow lists to override settings: {allow_override}\n"
+                "Use Spoilers in answers: {use_spoilers}"
+            ).format(**config_settings)
```

Comment:
This needs to be modified so that each line about the configuration only gets added if the appropriate override is present.

One way to do this would be to create a dictionary mapping key names to translated strings with a common placeholder (e.g. `{value}`) and formatting the value into them.

Just to mention it, `config_settings` is no longer defined *but*, that should get resolved once you make the change requested here.